code/_onclick/hud/rendering/plane_master.dm

```diff
@@ -577,6 +577,30 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/plane_master)
 	plane = GHOST_PLANE
 	render_relay_planes = list(RENDER_PLANE_NON_GAME)
 
+/atom/movable/screen/plane_master/ghost/Initialize(mapload, datum/plane_master_group/home, offset)
+	. = ..()
+
+	var/mob/hudmod = home.our_hud.mymob
+	if(isliving(hudmod))
+		// living? can't click on ghosts unless dead
+		RegisterSignal(hudmod, COMSIG_MOB_STATCHANGE, PROC_REF(handle_stat_change))
+		handle_stat_change(hudmod)
+
+	return .
```

Comment:
technically unneeded, user's preference. only need to explicit return . if you modify it inside the proc